.github/scripts/bundle.sh

```diff
@@ -13,25 +12,19 @@ export FIREBASE_PROJECT_ID=""
 export FIREBASE_STORAGE_BUCKET=""
 export INFURA_BETA_PROJECT_ID=""
 export INFURA_FLASK_PROJECT_ID=""
-export INFURA_MMI_PROJECT_ID=""
 export INFURA_PROD_PROJECT_ID=""
 export INFURA_PROJECT_ID=""
-export MMI_CONFIGURATION_SERVICE_URL=""
 export PUBNUB_PUB_KEY=""
 export SEGMENT_BETA_WRITE_KEY=""
 export SEGMENT_FLASK_WRITE_KEY=""
-export SEGMENT_MMI_WRITE_KEY=""
-export SEGMENT_PROD_LEGACY_WRITE_KEY=""
 export SEGMENT_PROD_WRITE_KEY=""
 export SEGMENT_WRITE_KEY=""
 export ANALYTICS_DATA_DELETION_SOURCE_ID=""
 export ANALYTICS_DATA_DELETION_ENDPOINT=""
 export SENTRY_AUTH_TOKEN=""
 export SENTRY_DSN=""
-export SENTRY_MMI_DSN=""
 export SENTRY_DSN_DEV=""
 export VAPID_KEY=""
-export TZ="UTC"
```

Comment:
Why was this removed?

Reply:
Yes. Firefox will reject our submission unless they can reproduce it exactly with the script we provide them, and we found through experimentation that the time zone does affect the build output somehow. I can't recall if we discovered why.